### PR TITLE
Add error and not found states

### LINKED ISSUE #837: Add Error and not-found states
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

Currently, there are no not-found and server error states. They should be added.

**Working on it**

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #837 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Add error and not found states

### CODE REVIEW

**Comment by ParthS007** [comment]
@djmgit  Please review :smiley:

File: frontend/app/templates/components/error-component/not-found.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<br>
+<h1 class="text muted weight-300" style="margin-bottom: 0;">404 Not Found</h1>
+<h1 style="font-size: 3.5em; " class="weight-300">Oops, the page you're<br> looking for does not<br> exist.</h1>
```
Comment (vaibhavsingh97) [inline frontend/app/templates/components/error-component/not-found.hbs]: same

File: frontend/app/templates/components/error-component/server-error.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<br>
+<h1 class="text muted weight-300" style="margin-bottom: 0;">500 Internal Server Error</h1>
```
Comment (vaibhavsingh97) [inline frontend/app/templates/components/error-component/server-error.hbs]: same

File: frontend/app/templates/components/error-component/server-error.hbs [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+<h1 style="font-size: 3.5em; " class="weight-300">Oops, the page you're<br> looking for caused an error<br>on our servers.</h1>
```
Comment (vaibhavsingh97) [inline frontend/app/templates/components/error-component/server-error.hbs]: same

File: frontend/app/templates/components/error-component/not-found.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<br>
+<h1 class="text muted weight-300" style="margin-bottom: 0;">404 Not Found</h1>
```
Comment (vaibhavsingh97) [inline frontend/app/templates/components/error-component/not-found.hbs]: please define CSS class instead of using inline CSS

**Pull request merged**